README.md

```diff
@@ -245,6 +245,28 @@ usage: grep_ioc KEYWORD [hutch]<br/>
     </td>
 </tr>
 
+<tr>
+    <td>grep_more_ioc</td>
+    <td>
+usage: grep_more_ioc [-h] [-d] patt hutch {print,search} <br/>
```

Comment:
The script is actually `grep_more_ioc.sh`, I think we should document as such here or rename the scripts to not have .sh suffixes.

Reply:
I'd vote to not have .sh as most scripts in engineering tools do not have extensions.

Reply:
To hop on to the readme suggestions, could you add an example call for those of us who don't know what pattern/hutch to search for?   I think examples are helpful for people not so familiar with EPICS and our systems

Reply:
I think if this script renaming/examples stuff get in then this PR has my approval (I don't want to hold it up any longer, but I'd like the names to be finalized + I'd like people to be ready to use it when it gets merged)